src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -322,6 +324,16 @@ internal void ToString(TraceFormat traceFormat, StringBuilder sb)
                             sb.Append(' ');
                             sb.AppendFormat(CultureInfo.InvariantCulture, inFileLineNum, fileName, sf.GetFileLineNumber());
                         }
+                        else
+                        {
+                            if (s_ilOffsetToStackTrace)
+                            {
+                                sb.Append(' ');
+                                string methodToken = mb.MetadataToken.ToString("x");
```

Comment:
`MetadataToken` is not always available and it can throw exceptions. 

For example, it is not available for DynamicMethods. This property throws `System.InvalidOperationException` on DynamicMethods.

Metadata tokens are also not available in some flavors of AOT.

Reply:
Thanks for review. I'll check those case and fix it